tensorbay/opendataset/UrbanObjectDetection/catalog.json

```diff
@@ -0,0 +1,21 @@
+{
+    "BOX2D": {
+        "attributes": [
+            {
+                "name": "pose",
+                "enum": ["Rear", null, "Left", "Right", "Unspecified", "none", "Frontal"]
```

Comment:
why we have `null` and `"none"`? Should them be the same?

Reply:
UrbanObjectDetection dataset has pose attributes like this, I guess that null,"none" and "unspecified" are the same